### DISCUSSION TITLE
How to improve the DevPool program

### DISCUSSION THREAD

**Discussion by sergfeldman** [ubiquity/.github#10] [DevPool]
Please, share your proposals on how to improve the DevPool program
You can suggest in this or other areas:
- Process automation
- Improvement of the Web3 project experience 
- Improvement of the Bounty hunter experience

**Comment by 0x4007** [comment]
> * Process automation

I was actually thinking to hire folks as "process automation engineers" from the DevPool once we begin to set up more functions of the organization. I wanted to focus first on our r&d processes first though.

**Comment by nicholasnokia** [comment]
2. Improvement of the Web3 project experience 

How about a gamefi experience ? 
Bounties based on difficulty level - stars to denote level of difficulty. 
Dapps like dota. Match making for devs to suitable projects based on skill levels. 

3. Once the profile of the devs have been set, the bot match make devs based on skills level. Devs start from 0 stars, climb ladder, and increase exp. pts. 
Overtime, the bot match makes / recommends suitable projects to devs.
